docs/MAINTAINING.md

```diff
@@ -164,13 +164,14 @@ If you're the current release manager, follow these steps:
 
 ### Publishing a release
 
-1. Create a new branch called `v0.0.0` (replace 0s with actual version aka v4.4.0)
+1. Create a new branch called `v0.0.0` (replace 0s with actual version aka v4.5.0)
 1. Run `yarn release:prep` and type in the new version (e.g., `3.8.1`)
 1. GitHub Actions will generate the `npm-package`, `release-packages` and
    `release-images` artifacts. You do not have to wait for this step to complete
    before proceeding.
 1. Run `yarn release:github-draft` to create a GitHub draft release from the
-   template with the updated version.
+   template with the updated version. Make sure to update the `CHANGELOG.md`.
+1. Bump chart version in `Chart.yaml`.
```

Comment:
lot of step 1s here 😄

Reply:
Ah yes, that's intentional! That way if you re-order, you don't have to go renumber them.

 🖼️ [rendered on GitHub](https://github.com/coder/code-server/blob/main/docs/MAINTAINING.md#publishing-a-release)

[Source](https://github.github.com/gfm/#list-items)

Reply:
TIL! Thanks for the link!